source/scripts/modules/Keys.js

```diff
@@ -13,19 +13,12 @@ window.uiGradients = window.uiGradients || {};
 
       $(document).on('keydown', function(event) {
         switch(event.which) {
-
+          
           case 37: // left
-            if (pallete.hasClass('is-active')) {
-              return;
-            }
-            canvas.trigger('changeDown');
-          break;
-
           case 39: // right
-            if (pallete.hasClass('is-active')) {
-              return;
-            }
-            canvas.trigger('changeUp');
+            if (pallete.hasClass('is-active')) return;
+
+            (event.which === 37 ) ? canvas.trigger('changeDown') : canvas.trigger('changeUp');
```

Comment:
After reading more about it, I was thinking maybe it is better to leave it as, for the sake of simplicity.

While I see the benefits, in this particular case having an if statement within kind of defeats the purpose. I would rather duplicate one line of code and preserve readability.

Anyway, thanks for introducing fallthrough to me, I am definitely going to use it to clean up some contrived if-else statements of mine. Live and learn!